apps/microtvm/reference-vm/arduino/provision_setup.sh

```diff
@@ -28,9 +28,9 @@ apps/microtvm/reference-vm/rebuild-tvm.sh ${platform}
 # Build poetry
 cd apps/microtvm/reference-vm/arduino
 
-poetry env use 3.6
+poetry env use 3.7
 # NOTE: due to https://github.com/python-poetry/poetry/issues/2247, download torch here.
```

Comment:
since we are checking in poetry.lock now (imo the right thing to do), shall we rm `poetry lock -vvv` below?

Reply:
yep.